ratatool-sampling/src/test/scala/com/spotify/ratatool/samplers/BigSamplerTest.scala

```diff
@@ -19,14 +19,13 @@ package com.spotify.ratatool.samplers
 import java.io.File
 import java.nio.ByteBuffer
 import java.nio.file.{Files, Path}
-
```

Comment:
Heavy Nitpick: I saw a lot of these removed. Most of us use IntelliJ which I think will just re-add this line on another auto-import. Can you re-add? I would understand if there was some pushback on this haha

Reply:
No worries. Re-added to lower the amount of diffs.